app/helpers/application_helper.rb

```diff
@@ -1,6 +1,7 @@
 # Methods added to this helper will be available to all templates in the application
 module ApplicationHelper
   include HomeHelper
+  include AuthorizeHelper
```

Comment:
This is not going to work in the current configuration.
`AuthorizeHelper` has `authorized_for` that is used in `ApplicationController` and also has `authorizer` which comes from `ApplicationHelper`. Including `AuthorizeHelper` only in one of those will probably cause problems as it is assumed that those methods (`authorized_for` and `authorizer`) are placed in `ApplicationController` and `ApplicationHelper` respectively. Also see that [this test](https://ci.theforeman.org/job/test_develop_pr_core/19414/database=postgresql,ruby=2.4,slave=fast/testReport/junit/(root)/HostGroupsHelperTest/test_0001_should_have_the_full_string_of_the_parent_class_if_the_child_is_a_substring/) and (this test)[https://ci.theforeman.org/job/test_develop_pr_core/19414/database=mysql,ruby=2.5,slave=fast/testReport/junit/(root)/Api__V2__FactValuesControllerTest/test_0006_should_search_facts_by_location/] (https://ci.theforeman.org/job/test_develop_pr_core/19414/#showFailuresLink)[failed]

I do agree I need to refactor the code base to make this happen. However, it will take time and be hard to review.

Reply:
Ah ok, ApplicationHelper is not included in ApplicationController. Funny. Ok, scratch that.